test/HostActivationTests/GivenThatICareAboutMultilevelSDKLookup.cs

```diff
@@ -230,7 +230,7 @@ public void SdkLookup_Must_Look_For_Available_Versions_Before_Looking_Into_Anoth
                 .Should()
                 .Pass()
                 .And
-                .HaveStdErrContaining(Path.Combine(_cwdSelectedMessage, "9999.0.0-global-dummy"));
+                .HaveStdErrContaining(Path.Combine(_cwdSelectedMessage, "9999.0.0"));
```

Comment:
Should we add a test where we add a newer preview SDK and see that it gets picked over the release one?

Like, have 9999.0.0 and 10000.0.0-preview1-something and the 10000.00 being the one picked up?